modules/standard/IO.chpl

```diff
@@ -4398,6 +4401,19 @@ proc channel.close() {
   if e then this._ch_ioerror(e, "in channel.close");
 }
 
+// TODO -- we should probably have separate c_ptr ddata and ref versions
```

Comment:
Michael, what would these separate versions do? Perhaps these comments should explain the deficiency of the current version(s).